src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterpriseUpload.scala

```diff
@@ -35,7 +36,15 @@ class TaskEnterpriseUpload(config: Configuration, enterprisePackage: TaskEnterpr
     val enterprisePlugin = EnterprisePluginTask.batchEnterprisePluginTask(config).value
 
     Try {
-      if (settingPackageId.isEmpty && settingSimulationId.isEmpty) {
+      val packageId = Option(PackageConfiguration.loadToJson(baseDirectory.value))
```

Comment:
Is baseDirectory the root of the project or the module?

Reply:
Depending on the scope :rofl: 
(by default, the "sbt" project, ie the submodule)

Reply:
Just in case, I don't test this code for now, it's just a draft. To be honest, I pick baseDirectory because the name sounds coherent :shrug: 
I will do some testing this afternoon.

Reply:
As @Isammoc said, the doc in Keys.scala is : 
> "The base directory.  Depending on the scope, this is the base directory for the build, project, configuration, or task."